utils/payment-term.js

```diff
@@ -137,14 +131,8 @@ class PaymentTerm {
 			if (trialPrice) {
 				trialPrice.innerHTML = update.trialPrice;
 			}
-			if (monthlyPrice) {
-				monthlyPrice.innerHTML = this.isValidPeriod(update.value)
-					? this.getMontlyPriceFromPeriod(
-							update.amount,
-							update.currency,
-							update.value
-					  )
-					: update.monthlyPrice;
+			if (monthlyPrice && update && update.monthlyPrice) {
```

Comment:
In line 120 we cover if Update doesn't exist: 
```suggestion
			if (monthlyPrice &&  update.monthlyPrice) {
```